test/integration/files/FilesRepository.test.ts

```diff
@@ -87,7 +87,14 @@ describe('FilesRepository', () => {
         throw new Error(`Tests beforeAll(): Error while uploading file ${testTabFile4Name}`)
       })
     // Registering test file 1
+
     await registerFileViaApi(uploadFileResponse.data.data.files[0].dataFile.id)
+      .then()
+      .catch((err) => {
+        console.log(err.message)
+        console.log(err.response.data)
+        throw new Error(`Tests beforeAll(): Error while registering file ${testTextFile1Name}`)
+      })
```

Comment:
The then() is not necessary since it's not performing any action on the promise. We can also remove the console.logs
```suggestion
    await registerFileViaApi(uploadFileResponse.data.data.files[0].dataFile.id)
      .catch((err) => {
        throw new Error(`Tests beforeAll(): Error while registering file ${testTextFile1Name}`)
      })
```

Reply:
thanks @MellyGray,I made the suggested changes